seed/templates/seed/_header.html

```diff
@@ -20,16 +31,22 @@
                 <i class="fa fa-plus"></i>
                 <span class="sr-only" translate>Toggle Dropdown</span>
             </button>
-            <ul uib-dropdown-menu class="dropdown-menu pull-right" role="menu" aria-labelledby="btnCreateNew">
-                <li class="dropdown-header" translate>Create a new...</li>
+            <ul uib-dropdown-menu class="dropdown-menu pull-right justify-left" role="menu" aria-labelledby="btnCreateNew">
+                <li class="dropdown-header" translate>Organization Settings</li>
                 <li class="divider"></li>
+                <li><a ui-sref="organization_settings({organization_id: org.id})" ng-if="auth.requires_owner" ui-sref-active="active" translate>Settings</a></li>
+                <li><a ui-sref="organization_sharing({organization_id: org.id})" ng-if="org.is_parent && auth.requires_owner" ui-sref-active="active" translate>Sharing</a></li>
+                <li><a ui-sref="organization_column_settings({organization_id: org.id, inventory_type: 'properties'})" ng-if="auth.requires_owner" ui-sref-active="active" translate>Column Settings</a></li>
+                <li><a ui-sref="organization_data_quality({organization_id: org.id, inventory_type: 'properties'})" ng-if="org.is_parent && auth.requires_owner" ui-sref-active="active" translate>Data Quality</a></li>
+                <li><a ui-sref="organization_cycles({organization_id: org.id})" ng-if="org.is_parent && auth.requires_owner" ui-sref-active="active" translate>Cycles</a></li>
+                <li><a ui-sref="organization_labels({organization_id: org.id})" ng-if="auth.requires_owner" ui-sref-active="active" translate>Labels</a></li>
+                <li><a ui-sref="organization_members({organization_id: org.id})" ui-sref-active="active" translate>Members</a></li>
+                <li><a ui-sref="organization_members({organization_id: org.id})" translate>More...</a></li>
```

Comment:
`More...` sends users to `.../members`. This was pulled from the organization link in `.../accounts`